src/routes/v2/projects.rs

```diff
@@ -2558,3 +2561,134 @@ pub async fn delete_from_index(
+    let versions = db_models::Version::get_many(&project_data.versions, &**pool, &redis).await?;
+    let mut versions = filter_authorized_versions(versions, &None, &pool).await?;
+    versions.sort_by(|a, b| b.date_published.cmp(&a.date_published));
+    let authors = users
+        .into_iter()
+        .map(|u| atom_syndication::Person {
+            uri: Some(format!("{site_url}/user/{}", &u.username)),
+            name: u.username,
+            email: None,
+        })
+        .collect::<Vec<_>>();
+
+    let project = models::projects::Project::from(project_data);
+    let mut feed = atom_syndication::Feed::default();
+    feed.title = atom_syndication::Text::plain(project.title.to_string());
+    feed.links.push(atom_syndication::Link {
+        href: format!("{site_url}/mod/{string}"),
+        rel: "alternate".to_string(),
+        mime_type: Some("text/html".to_string()),
+        title: Some(project.title.to_string()),
+        hreflang: None,
+        length: None,
+    });
+    feed.links.push(atom_syndication::Link {
+        href: format!("{self_addr}/project/{string}/feed"),
+        rel: "self".to_string(),
+        mime_type: Some("application/atom+xml".to_string()),
+        title: Some(project.title.to_string()),
+        hreflang: None,
+        length: None,
+    });
+    feed.icon = project.icon_url;
+    feed.id = format!("{site_url}/mod/{string}");
+    feed.updated = project.updated.fixed_offset();
+    feed.rights = Some(project.license.name.into());
+    for category in &project.categories {
+        feed.categories.push(atom_syndication::Category {
+            term: category.to_string(),
+            label: Some(category.to_string()),
+            scheme: None,
+        });
+    }
+    for version in versions {
+        let mut entry = atom_syndication::Entry::default();
+        let link = format!("{site_url}/mod/{string}/version/{}", version.version_number);
+        entry.id = link.clone();
+        entry.title = atom_syndication::Text::plain(version.name.to_string());
+        entry.updated = version.date_published.fixed_offset();
+        entry.published = Some(version.date_published.fixed_offset());
+        let rendered_changelog = comrak::markdown_to_html(&version.changelog, &markdown_options);
+
+        entry.content = Some(atom_syndication::Content {
+            base: Some(link.clone()),
+            content_type: Some("html".to_string()),
+            lang: None,
+            src: None,
+            value: Some(rendered_changelog.to_string()),
+        });
+        entry.links.push(atom_syndication::Link {
+            href: link.clone(),
+            rel: "alternate".to_string(),
+            mime_type: Some("text/html".to_string()),
+            title: Some(version.name.to_string()),
+            hreflang: None,
+            length: None,
+        });
+        entry.authors = authors.clone();
+        feed.entries.push(entry);
```

Comment:
This should probably also include the game version/loader metadata somewhere, as from what I can tell if you have duplicate version numbers there really isn't any good way to figure out for what loader it is

Reply:
I've added the compatible game versions and loaders as categories. Here's what it looks like in Miniflux:
![image](https://github.com/modrinth/labrinth/assets/1643883/e5a498bd-ea1c-4925-b71f-6f0ef97e220d)

Reply:
Amazing! ~~did the RSS readers i tested my stuff with not support categories~~